region/lowernorfair/east/Lower Norfair Spring Ball Maze Room.json

```diff
@@ -194,6 +195,58 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 5],
+      "name": "Blue Spacejump Through Morph Tunnel (Come in Jumping)",
+      "entranceCondition": {
+        "comeInBlueSpinning": {
+          "minExtraRunSpeed": "$6.E",
+          "unusableTiles": 0
+        }
+      },
+      "requires": [
+        {"notable": "Air Speedball"},
+        "canPreciseSpaceJump",
+        "canInsaneJump",
+        "canLateralMidAirMorph",
+        "canSpeedball",
+        "canMomentumConservingMorph",
+        {"heatFrames": 110}
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Spacejump through the room and break the bomb blocks in the Morph tunnel with Speedbooster.",
+        "It is very helpful for the final jump to be close to the missile platform, to be as close to the tunnel as possible when Morphing."
+      ],
+      "devNote": [
+        "This is possible with less extra speed but where you jump changes with your speed and room entry."
+      ]
+    },
+    {
+      "link": [1, 5],
+      "name": "Blue Spacejump Through Morph Tunnel (Come in Running)",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 5,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$5.B"
```

Comment:
I'm surprised the `minExtraRunSpeed` here isn't closer to the value in the other strat; is it maybe a mistake? Note that `minExtraRunSpeed` represents the minimum speed attainable at the end of the runway, not room entry.